src/app/components/client/toolbar/AnnouncementDialog.module.scss

```diff
@@ -91,6 +98,13 @@
     overflow-y: auto;
 
     .announcementItem {
+      background: none;
+      background-color: none;
```

Comment:
I'm not sure this is a valid value for `background-color` https://developer.mozilla.org/en-US/docs/Web/CSS/background-color